index.html

```diff
@@ -101,7 +101,9 @@ <h1 class='h1-h'>Porsche's Taycan Cross Turismo EV isn't as confusing as it look
     <!-- Latest News -->
     <section class="news">
```

Comment:
## Improve the general structure of the page
- [ ] The general structure of the page from top to bottom are not aligned as viewed or compared with the original page.  You should note that the first main three articles must not cover the same width as that of the rest of the contents of the page. The width is much wider in this section as the rest of the section gets to be uniform, lesser and centred. have a look at the comparison below for inspiration.
- [ ] Be mindful of the spacing around elements and the font sizes as well.they appear much bolder in the source page.

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/MarkoNS1990/the_next_web/milestone-2/index.html)|
|:--:|:--:|
|![latest_deals](https://user-images.githubusercontent.com/22345961/109025467-ce840b00-76cf-11eb-9c5e-045f724f8b8b.png)|![latest_deals_clone](https://user-images.githubusercontent.com/22345961/109025508-d774dc80-76cf-11eb-9391-68efcf565a88.png)|